gateway-service/service.bal

```diff
@@ -0,0 +1,161 @@
+// Copyright (c) 2023, WSO2 LLC. (http://www.wso2.org) All Rights Reserved.
+
+// WSO2 LLC. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+
+//    http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied. See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/http;
+import ballerina/lang.regexp;
+import ballerina/log;
+import ballerinax/azure.openai.chat;
+import ballerinax/azure.openai.embeddings;
+import ballerinax/azure.openai.text;
+
+configurable string azureOpenAIToken = ?;
+configurable string azureOpenAIServiceUrl = ?;
+configurable string azureOpenAIApiVersion = ?;
+
+final text:Client textClient = check new (
+    config = {auth: {apiKey: azureOpenAIToken}},
+    serviceUrl = azureOpenAIServiceUrl
+);
+
+final chat:Client chatClient = check new (
+    config = {auth: {apiKey: azureOpenAIToken}},
+    serviceUrl = azureOpenAIServiceUrl
+);
+
+final embeddings:Client embeddingsClient = check new (
+    config = {auth: {apiKey: azureOpenAIToken}},
+    serviceUrl = azureOpenAIServiceUrl
+);
+
+const string PII_DETECTION_ERROR_MESSAGE = "PII detected in the request payload";
+
+# A service representing a network-accessible API
+# bound to port `9090`.
+service / on new http:Listener(9090) {
+
+    resource function post [string deploymentId]/completions(@http:Payload text:Deploymentid_completions_body request) returns text:Inline_response_200|http:BadRequest|error {
```

Comment:
`text:Inline_response_200` shouldn't it be `text:InlineResponse200`?

Reply:
Usual practise is to use PascalCase.

Reply:
@shafreenAnfar this is a type defined in the connector which was generated by the ballerina openapi tool